files/quorum-key-manager/manifests/vaults.yml.template

```diff
@@ -0,0 +1,45 @@
+- kind: Vault
+  type: hashicorp
+  name: hashicorp-kv-v2
+  # allowed_tenants: [tenant1, tenant2]
+  specs:
+    mount_point: secret
+    address: http://hashicorp:8200
+    token_path: /vault/token/.root
+    namespace: ''
+    # client_cert: /certificates/client.crt
+    # client_key: /certificates/client.key
+    # ca_cert: /ca/ca.crt
+
+- kind: Vault
+  type: hashicorp
+  name: hashicorp-quorum
+  # allowed_tenants: [tenant1, tenant2]
+  specs:
+    mount_point: quorum
+    address: http://hashicorp:8200
+    token_path: /vault/token/.root
+    namespace: ''
+    # client_cert: /certificates/client.crt
+    # client_key: /certificates/client.key
+    # ca_cert: /ca/ca.crt
+
+- kind: Vault
```

Comment:
do we want to keep the aws and akv stores in the quickstart? If so, why?

Reply:
this is intended to be useful an templating in case clients want to use it